client/my-sites/checkout/upsell-nudge/index.jsx

```diff
@@ -552,14 +555,17 @@ const trackUpsellButtonClick = ( eventName ) => {
 	return recordTracksEvent( eventName, { section: 'checkout' } );
 };
 
-const resolveProductSlug = ( upsellType, productAlias ) => {
+const resolveProductSlug = ( upsellType, productAlias, billingTerm ) => {
```

Comment:
Since we actually pass a `currentPlanTerm` when we call that function a few lines below we could use the same term for that parameter (so it's consistent and also makes it clear that this information is specific to the plan chosen):

```suggestion
const resolveProductSlug = ( upsellType, productAlias, planTerm ) => {
```